### ISSUE TITLE
Problems with Discord login button

### ISSUE THREAD

**Issue by tw1tterf0rblackb3rry** [binary-person/rammerhead#20]
There is a problem where eventually, trying to login to discord does not work. When clicking on the login button nothing happens and it seems like you have to wait about 7, 6, or 8 hours before the login button actually works. this started happening yesterday and it has been very frustrating since I use discord for school. please if you can, fix this issue..

**Comment by binary-person** [comment]
is this still happening?

**Comment by tw1tterf0rblackb3rry** [comment]
No, this is no longer happening but as of today, the login screen won't load at all. Also, apologies for the very later response.

**Comment by 3kh0** [comment]
@tw1tterf0rblackb3rry any updates?
